spyplane/services/post_after_tick_service.py

```diff
@@ -27,12 +31,22 @@ async def run_after_interval(self):
             log("Posting systems now")
             await SystemsPostingService().publish_systems_to_scout()
         except Exception as e:
-            log("Exception when running after interval")
-            log(str(e))
+            log_exception("run_after_interval", e)
 
-    async def validate_message(self, message: discord.message.Message):
-        # if 'Tick Detected' not in message and 'Latest Tick At' not in message:
-        #     print("'Tick Detected' and 'Latest Tick At' is not in message")
-        #     return
+    async def tick_check_and_schedule(self):
```

Comment:
Unused, along with the 2 new files below. 
Left it here for reference, or if the BGS bot based one fails in future.